e2e.groovy

```diff
@@ -0,0 +1,59 @@
+def cico_retries = 16
+def cico_retry_interval = 60
+def ci_git_repo = 'https://github.com/Yuggupta27/ceph-csi'
+def ci_git_branch = 'e2e-centosci'
+def ref = "master"
+
+node('cico-workspace') {
+
+	stage('checkout ci repository') {
+		git url: "${ci_git_repo}",
+			branch: "${ci_git_branch}",
+			changelog: false
+	}
+
+	stage('reserve bare-metal machine') {
+		def firstAttempt = true
+		retry(30) {
+			if (!firstAttempt) {
+				sleep(time: 5, unit: "MINUTES")
+			}
+			firstAttempt = false
+			cico = sh(
+				script: "cico node get -f value -c hostname -c comment --retry-count ${cico_retries} --retry-interval ${cico_retry_interval}",
+				returnStdout: true
+			).trim().tokenize(' ')
+			env.CICO_NODE = "${cico[0]}.ci.centos.org"
+			env.CICO_SSID = "${cico[1]}"
+		}
+	}
+	try {
+		stage('prepare bare-metal machine') {
+			if ("${ghprbPullId}".length() != 0) {
+				ref = "pull/${ghprbPullId}/head"
+			}
+			sh 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ./prepare.sh ./multi-node-k8s.sh root@${CICO_NODE}:'
+			sh "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} ./prepare.sh --workdir=/opt/build/go/src/github.com/ceph/ceph-csi --gitrepo=${ci_git_repo} --ref=master"
+		}
+
+		stage('deploy kubernetes') {
+			sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} ./multi-node-k8s.sh'
+		}
+
+		stage('deploy rook') {
+			//sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} virsh reboot k8s-vagrant-multi-node_node2; sleep 120'
+			sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} /opt/build/go/src/github.com/ceph/ceph-csi/scripts/rook.sh deploy'
+		}
+
+		stage('e2e') {
+			sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} "make -C /opt/build/go/src/github.com/ceph/ceph-csi containerized-build TARGET=e2e.tests"'
```

Comment:
This command errors like this:
```
podman run --rm -v /root:/go/src/github.com/ceph/ceph-csi:z quay.io/cephcsi/cephcsi:devel make cephcsi
Error: relabel failed "/root": SELinux relabeling of /root is not allowed
make: *** [containerized-build] Error 126
```

You can work around it by `cd /opt/build/go/src/github.com/ceph/ceph-csi && make container....`

Reply:
The problem is that the `Makefile` uses `$(PWD)` with the `$(CONTAINER_CMD) run` commands. This gets replaced by the current working directory, but should be the 'current `make` directory' using `$(CURDIR)`.

Reply:
See #1122 for a fix